tests/baselines/reference/commentsArgumentsOfCallExpression2.js

```diff
@@ -18,6 +18,6 @@ foo(/*c3*/ function () { }, /*d2*/ function () { }, /*e2*/ a + b);
 foo(/*c3*/ function () { }, /*d3*/ function () { }, /*e3*/ (a + b));
 foo(
 /*c4*/ function () { }, 

```

Comment:
Edit : I accidentally delete my original comment.
For record: the old comment: is there trailing whitespace here? If so, should this change remove it?

@MartyIX  yes fixing all trailing spaces in all scenario are harder problem.